src/index.js

```diff
@@ -10,12 +10,14 @@ const {
   help = defaultOptions.help,
   h = defaultOptions.help,
   discardComments = defaultOptions.discardComments,
+  check = defaultOptions.check,
 } = minimist(process.argv.slice(2));
 
 if (help || h) {
```

Comment:
Just out of curiosity, Why help and h? Is it because you accept both `--help` plus the short version `-h`?

Reply:
it is a common pattern to support the short form `-h` along with `--help`